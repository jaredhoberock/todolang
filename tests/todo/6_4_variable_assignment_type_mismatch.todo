var x: String = "Hello";
x = 13;
print x;
