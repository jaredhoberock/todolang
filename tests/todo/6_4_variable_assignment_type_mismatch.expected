Type mismatch: expected 'String', found 'Number'
